pkg/network/ebpf/c/shared-libraries/probes.h

```diff
@@ -144,4 +145,13 @@ int tracepoint__syscalls__sys_exit_openat2(exit_sys_ctx *args) {
     return 0;
 }
 
+SEC("fexit/do_sys_openat2")
+int BPF_BYPASSABLE_PROG(do_sys_openat2_exit, int dirfd, const char *pathname, void *how, long ret) {
```

Comment:
I think this will make the tests added by https://github.com/DataDog/datadog-agent/pull/32402 to fail since this prog doesn't check for the flags? In that PR, open(2) and openat(2) were supported while openat2(2) was not, but after this PR, all three system calls are handled via this hook point on relevant kernels.

Reply:
added support to ignore write-only calls